File: Visual Studio Code/Practicas/Ejercicio 3/promedionotas.js
for (let i = 0; i <  6; i++) {
    nota = parseFloat(prompt("Ingrese su calificacion: "))
    if (nota <0 && nota <= 5)
        notas = nota / 5
    if (nota >= 3)
        console.log("Usted aprobo. ")
    else
        console.log("Usted reprobo")
    }
console.log("Su promedio fue", nota)

File: Practicas/Ejercicio 3/promedioNotas.py
for i in range (6):
    nota = float(input("ingrese su calificacion: "))
    if nota <0 and nota >= 5:
        notas = nota / 5
        break
    if nota >= 3:
        print("usted aprobo")
    else:
        print("usted reprobo")
print("su promedio final fue: " + nota)